be/src/exec/pipeline/olap_scan_operator.h

```diff
@@ -4,10 +4,10 @@
 
 #include "exec/pipeline/pipeline_builder.h"
 #include "exec/pipeline/scan_operator.h"
-#include "exec/scan_node.h"
```

Comment:
Is this forward declaration automatic? Or use some tool?

Reply:
By hand :(